week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,16 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+function mentors(name) {
+  return name.toUpperCase(name);
+}
+function greeting(name) {
+  return "HELLO  " + mentors(name);
+}
+console.log(greeting(mentor1));
+console.log(greeting(mentor2));
+console.log(greeting(mentor3));
+console.log(greeting(mentor4));
+console.log(greeting(mentor5));
```

Comment:
It's a nice idea to create a separate file for the other exercise!

Another programming tip, is that usually, we avoid jumping to too many functions if possible. For example, you can make the `name` uppercase in the `greeting()` function, and you can get rid of the `mentors` function - how can you do that?

Reply:
Hi martyna Sorry I rewrite the code in a single file .Kindly check it for me please.

Reply:
I have changed it.Thanks martyna